docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 breathe==4.22.1
```

Comment:
Not related to this file.
(I did not find a way to add a comment not related to a files in an open review)

Did you notice that when a tab is changed ALL tab components are affected and the current position in the document is lost?

It affect all tabs with the same label in the page.
```
function onLabelClick() {
  // Activate other labels with the same text.
  for (label of labels_by_text[this.textContent]) {
    if (label === this) continue;
    label.previousSibling.checked = true;
  }
}
```

Reply:
>Did you notice that when a tab is changed ALL tab components are affected

Yes, this is actually a feature all tabs with the same name are syncronized.

> and the current position in the document is lost?

This is an, unfortunately, a side effect  =/ 
Since different tabs can have different outputs that affect the page height, this difference is more noticeable at the bottom.